projects/lookup/src/modules/autocomplete/autocomplete.component.ts

```diff
@@ -282,8 +283,12 @@ export class SkyAutocompleteComponent
     return this._searchResults || [];
   }
 
-  public get highlightText(): string {
-    return this._highlightText || '';
+  public set highlightText(value: string[]) {
+    this._highlightText = value;
+  }
+
+  public get highlightText(): string[] {
```

Comment:
It looks like `[skyHighlight]` only takes a `string`, but we're passing in an array of strings? I'm confused how this is working... 😅

Reply:
https://github.com/blackbaud/skyux-indicators/pull/198 😄